Gems/AtomTressFX/Assets/Passes/HairResolvePPLL.pass

```diff
@@ -139,6 +139,7 @@
             ],
             "PassData": {
                 "$type": "FullscreenTrianglePassData",
+                "PipelineViewTag": "MainCamera",
```

Comment:
I'm pretty sure this is not required since it takes directly from PPLL buffers and no other feed of the pipeline

Reply:
This is required since this pass's shader uses ViewSrg. 
It actually reported an error (RHI report missing viewSrg) for one frame without this PipelineViewTag. Then the error was gone in followed frame because the viewSrg was setup in other pass got carried over (using the same command list) for this pass.